spec/lib/vanagon/driver_spec.rb

```diff
@@ -54,6 +54,40 @@ def create_driver(platform, options = {})
                               'engine' => 'hardware' })
     end
 
+    it 'returns the platform and a scheduler engine when scheduled resources are enabled and an engine is specified' do
+      ENV['VANAGON_SCHEDULED_HARDWARE'] = "true"
+
+      platform = eval_platform('aix-7.1-ppc', <<-END)
+        platform 'aix-7.1-ppc' do |plat|
+          plat.build_host ["pe-aix-71-01", "pe-aix-71-02"]
+          plat.engine "scheduled"
+        end
+      END
+
+      info = create_driver(platform).build_host_info
+
+      expect(info).to match({ 'name'   => 'aix-7.1-ppc',
```

Comment:
Earlier we were talking about being consistent in our naming scheme for ABS, and using vmpooler as the gold standard for better or worse. It doesn't use dotted version numbers, e.g. `osx-1011-x86_64`, so I'm thinking this should be `aix-71-ppc`, which I think is worse, but would at least be consistently worse.